### DISCUSSION TITLE
People of WordPress scheduling on GitHub

### DISCUSSION THREAD

**Discussion by abhansnuk** [wpmarketingteam/Marcomms-Series#25] [Announcements]
## 👋 Welcome to the discussion on how we use GitHub for the People of WordPress scheduling and updates for the Team
 Things for us to consider:
- how we name the issues to make it easier to change the date when publications are rescheduled? This also needs to be clear to be able to use the links to GitHub in a Marketing Team sprint or meeting with calls for volunteers
- from the previous discussions: Should we also be linking to them in the Agenda document or to the general call for volunteers for the whole series which we could make into a wiki page with the skills required?
- how far in advance do we enter features from the schedule on GitHub (schedule currently goes in 2022)?
- how do we avoid the names appearing in general Google search results?
- do we just keep the link to the Google working document in the planning schedule rather than in GitHub especially before the fact-checking editorial phase?
- where do we add search and cross-links to be used, in the issue itself?
- where do we add links to the original and edited picture files? Just in the spreadsheet rather than in GitHub as these may be the original pictures. Credit information - where is this stored?
- is this series an example which could be used for the new document workflow process being discussed with training?

**Comment by abhansnuk** [comment]
Also welcome your thoughts on this so we can take it forward in the next GitHub and People of WordPress session in the Marketing Channel. @OlaIola @murillol @meherbala @femkreations @yvettesonneveld @MaedahBatool @yashwardhanrana @TantienHime 
(do we have Alison's GitHub to include her please)

**Comment by murillol** [comment]
**How we name the issues to make it easier to change the date when publications are rescheduled?**  I think that **month-year** (e.g. July 2021) is acceptable, because every month we need to publish _something_. Then, within the issue/card itself, we can change the details of which person is being featured. 

**Should we also be linking to them in the Agenda document or to the general call for volunteers for the whole series which we could make into a wiki page with the skills required?** Agree we need a wiki, not sure about the general call for volunteers (given our expriences so far, do we have the bandwidth to onboard new people on to this particular task?)

**How far in advance do we enter features from the schedule on GitHub (schedule currently goes in 2022)**? I think we should have a view at least 6 months into the future 👯 

**How do we avoid the names appearing in general Google search results?** While this stuff may be indexed, it would probably take someone searching for an exact match phrase for any of these to turn up in search results. However I will double check about this and get back to you.

**Reply by abhansnuk** [reply]
An update on the schedule has been added to the Macro card for this series https://github.com/wpmarketingteam/Marcomms-Series/issues/29
